### PR TITLE
Bump lockfile for 8.4.1

### PR DESCRIPTION
**Pull request by robbavey**

### CODE REVIEW

File: Gemfile.jruby-2.6.lock.release [hunk 1]
```diff
@@ -51,7 +51,7 @@ GEM
     aws-sdk-cloudwatch (1.65.0)
       aws-sdk-core (~> 3, >= 3.127.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-core (3.131.5)
+    aws-sdk-core (3.131.6)
```
Comment (yaauie) [inline Gemfile.jruby-2.6.lock.release:54]: ✅ https://github.com/aws/aws-sdk-ruby/blob/version-3/gems/aws-sdk-core/CHANGELOG.md#31316-2022-08-03

File: Gemfile.jruby-2.6.lock.release [hunk 2]
```diff
@@ -785,9 +785,9 @@ GEM
       multipart-post (~> 2.0)
       naught (~> 1.0)
       simple_oauth (~> 0.3.0)
-    tzinfo (2.0.4)
+    tzinfo (2.0.5)
```
Comment (yaauie) [inline Gemfile.jruby-2.6.lock.release:788]: ✅ https://github.com/tzinfo/tzinfo/blob/master/CHANGES.md#version-205---19-jul-2022

**Review by yaauie — APPROVED** [review]
All changes we are pulling in make sense and appear to be minimal. LGTM.

**Comment by yaauie** [comment]
Build failure caused by unreleased 8.4.1 (snapshot?) artifacts not being available because 8.4.0 release was today.

**Comment by robbavey** [comment]
Thank you for the thorough and thoughtful review ❤️

**Pull request merged**